src/pretix/control/views/orders.py

```diff
@@ -66,12 +66,29 @@ def get_queryset(self):
         if self.request.GET.get("item", "") != "":
             i = self.request.GET.get("item", "")
             qs = qs.filter(positions__item_id__in=(i,))
+        if self.request.GET.get("provider", "") != "":
+            p = self.request.GET.get("provider", "")
+            qs = qs.filter(payment_provider=p)
         return qs.distinct()
 
+    def get_payment_providers(self):
+        providers = []
+        responses = register_payment_providers.send(self.request.event)
+        for receiver, response in responses:
+            provider = response(self.request.event)
+            if not provider.is_enabled or not provider.is_allowed(self.request):
```

Comment:
One could probably argue that we should ignore ``is_enabled`` here as well, because even if a provider is (temporarily) disabled, there might be orders using that provider.

Providers from disabled plugin won't show up in the responses in the first place.